### PR TITLE
Changes for SimpleSAMLPHP 1.17+ compatibility

### PR DESCRIPTION
**Pull request by lionick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ This table matches the module version with the supported SimpleSAMLphp version.
 
 | Module |  SimpleSAMLphp |
 |:------:|:--------------:|
-| v1.0   | v1.14          |
+| v2.0   | v1.18          |
```
Comment (NicolasLiampotis) [inline README.md]: v1.x | v1.14
v2.x | v1.17+

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Backport bugfix from v1.14 release
Update version compatibility matrix

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Please rebase based on latest v1.x changes

**Comment by lionick** [comment]
> Please rebase based on latest v1.x changes

Done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**